src/components/ens-registration/PendingRegistrations/PendingRegistrations.tsx

```diff
@@ -0,0 +1,166 @@
+      setTimeout(() => {
+        navigate(Routes.ENS_CONFIRM_REGISTER_SHEET, {});
+      }, 100);
+    },
+    [navigate, startRegistration]
+  );
+
+  const onRemove = useCallback(
+    async (name: string) => {
+      removeRegistration(name);
+    },
+    [removeRegistration]
+  );
+
+  return (
+    <Box>
+      <Columns alignVertical="center">
+        <Column width="content">
+          <Box paddingRight="10px">
+            <ImageAvatar image={avatarUrl || ''} size="small" />
+          </Box>
+        </Column>
+        <Column>
+          <Box>
+            <Text color="primary" numberOfLines={1} size="16px" weight="heavy">
+              {registration.name}
+            </Text>
+          </Box>
+        </Column>
+        <Column width="content">
+          <Box paddingRight="15px">
+            <ButtonPressAnimation
+              onPress={() => onFinish(registration.name)}
+              scaleTo={0.9}
+            >
+              <Box
+                alignItems="center"
+                as={LinearGradient}
+                borderRadius={16}
+                colors={colors.gradients.transparentToAppleBlue}
+                end={{ x: 0.6, y: 0 }}
+                height="30px"
+                justifyContent="center"
+                start={{ x: 0, y: 0.6 }}
+              >
+                <Inset horizontal="10px">
+                  <AccentColorProvider color={colors.appleBlue}>
+                    <Text color="accent" size="16px" weight="heavy">
+                      Finish
+                    </Text>
+                  </AccentColorProvider>
+                </Inset>
+              </Box>
+            </ButtonPressAnimation>
+          </Box>
+        </Column>
+        <Column width="content">
+          <ButtonPressAnimation
+            onPress={() => onRemove(registration.name)}
+            scaleTo={0.9}
+          >
+            <Text color="secondary50" size="18px" weight="bold">
+              􀈒
+            </Text>
+          </ButtonPressAnimation>
+        </Column>
+      </Columns>
+    </Box>
+  );
+};
+
+const PendingRegistrations = () => {
+  const {
+    pendingRegistrations,
+    removeRegistrationByName,
+    registrationImages,
+  } = useENSPendingRegistrations();
+
+  const removeRegistration = useCallback(
+    (name: string) => {
+      Alert({
+        buttons: [
+          {
+            style: 'cancel',
+            text: lang.t('profiles.pending_registrations.alert_cancel'),
+          },
+          {
+            onPress: () => {
+              removeRegistrationByName(name);
+            },
+            text: lang.t('profiles.pending_registrations.alert_confirm'),
+          },
+        ],
+        message: lang.t('profiles.pending_registrations.alert_message'),
+        title: lang.t('profiles.pending_registrations.alert_title'),
+      });
+    },
+    [removeRegistrationByName]
+  );
+
+  return pendingRegistrations?.length ? (
+    <>
+      <Inset vertical="24px">
+        <Divider />
```

Comment:
hm the divider presence depends on `pendingRegistrations` if I move it up I'd need to use the hook in `ENSSearchSheet`, even in that case?

Reply:
Can we use Stack with separators? 

`<Stack separator={<MockDivider />} space="19px">`

This will render a divider dependant on if there is >1 child.

Reply:
sounds perfect 💯

Reply:
ah wait no, it's a divider on top of everything not in between elements